pkg/apis/numaflow/v1alpha1/const.go

```diff
@@ -122,6 +125,10 @@ const (
 	DefaultBufferUsageLimit = 0.8
 	DefaultReadBatchSize    = 500
 
+	// Edge-level specification to define the writing behaviour when the ToBuffer is full
```

Comment:
Minor - These definitions could stay in the edge_types.go.

Reply:
done.